packages/create-svelte/templates/default/package.json

```diff
@@ -14,7 +14,7 @@
 		"svelte": "^3.37.0",
 		"svelte-preprocess": "^4.7.1",
 		"typescript": "^4.2.4",
-		"vite": "2.2.2"
+		"vite": "^2.2.3"
```

Comment:
I think you need to update https://github.com/sveltejs/kit/blob/master/packages/create-svelte/templates/skeleton/package.template.json as well